base/applications/cmdutils/fc/lang/de-DE.rc

```diff
@@ -2,40 +2,40 @@ LANGUAGE LANG_GERMAN, SUBLANG_NEUTRAL
 
 STRINGTABLE
 BEGIN
-    IDS_USAGE "Compares two files or sets of files and displays the differences between\n\
-them\n\
+    IDS_USAGE "Vergleicht zwei Dateien oder Gruppen von Dateien und zeigt die\n\
+Unterschiede zwischen ihnen an\n\
```

Comment:
Shouldn't sentence end with full-stop? (english translation might have forgotten it, but anyways)

Reply:
Yes, Robert intended to stick to the english here obviously. I don't see anything wrong with that. Letting them diverge is most likely not better than keeping it as it is. Since it is even followed by 2 \n\n nobody will ever have a problem with not being able to understand that this is the end.

Reply:
> Robert intended to stick to the english here obviously

Not anymore :stuck_out_tongue_closed_eyes: